[PORT] 9.0 stock_picking_barcode (work in progress)

x620 opened the pull request.

Commit:
[ADD] Init commit: created manifest files - README.rst and __openerp__.py

Commit:
[DOC] Corrected summary and runbot link in the README.rst

Commit:
[PORT] Copied files which need to picking barcode

yelizariev commented on stock_picking_barcode/__openerp__.py:
```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+{
+    "name": "Stock Picking Barcode",
+    "summary": """Allow check products by barcode""",
```
```
 handle Pickings by barcode scanner
```

?

yelizariev commented on stock_picking_barcode/stock_report.xml:
```diff
@@ -0,0 +1,27 @@
+        <report id="action_report_location_barcode" model="stock.location" report_type="qweb-pdf" name="stock.report_location_barcode" string="Location BarCode" file="stock.report_location_barcode"/>
+        <report id="action_report_lot_barcode" model="stock.production.lot" report_type="qweb-pdf" name="stock.report_lot_barcode" string="Lot BarCode" file="stock.report_lot_barcode"/>
+    </data>
+</openerp>
```
delete it. It's already in stock module

Commit:
[PORT] Removed unused files

Commit:
[PORT] Added python functions

Commit:
[FIX] Fix errors which arise during the installation
[IMP] Added barcode scanner icon in the dashboard

Commit:
[FIX] Icon call Barcode scanner page

Commit:
[FIX] Open /barcode/web/ page

Commit:
[FIX] Send barcode to process_barcode_from_ui

Commit:
[FIX] Show rows in the table

Commit:
[FIX] Field lot_id was modificated from many2one to many2many so modificated code that rows added for each pack_lot_ids item
[FIX] Replaced processed field to peocessed_boolean
[FIX] Modificated picking.xml for show qty form in the product table
[FIX] Fix error search products - replaced 'product.template' to 'product.product' in the models.py

x620 commented:
We decided to maintain this module in our repository: https://github.com/it-projects-llc/misc-addons/pull/225
